src/EFCore.SqlServer/Extensions/SqlServerDbFunctionsExtensions.cs

```diff
@@ -16,6 +16,44 @@ namespace Microsoft.EntityFrameworkCore;
 /// </remarks>
 public static class SqlServerDbFunctionsExtensions
 {
+    /// <summary>
+    ///     Extracts a scalar value from a JSON string. <paramref name="propertyReference" /> and <paramref name="path" />.
+    ///     Corresponds to SQL Server's <c>JSON_VALUE(@propertyReference, @path)</c>.
+    /// </summary>
+    /// <remarks>
+    ///     See <see href="https://aka.ms/efcore-docs-database-functions">Database functions</see>, and
+    ///     <see href="https://aka.ms/efcore-docs-sqlserver">Accessing SQL Server and SQL Azure databases with EF Core</see>
+    ///     for more information and examples.
+    /// </remarks>
+    /// <param name="_">The <see cref="DbFunctions" /> instance.</param>
+    /// <param name="propertyReference">The property in table</param>
+    /// <param name="path">The json path.</param>
+    /// <returns>Value of path</returns>
```

Comment:
Copied reduced versions of the descriptions from the SQL Server docs, and added a link to them.
```suggestion
    /// <param name="expression">An expression. Typically the name of a variable or a column that contains JSON text.</param>
    /// <param name="path">A JSON path that specifies the property to extract.</param>
    /// <returns>A single text value.</returns>
    /// <seealso href="https://learn.microsoft.com/sql/t-sql/functions/json-value-transact-sql">SQL Server documentation for <c>JSON_VALUE</c>.</seealso>
```